features/step_definitions/ab_testing_steps.rb

```diff
@@ -13,7 +13,7 @@
 end
 
 Given(/^I am in the "(A|B)" group for "(.*)" AB testing$/) do |test_group, ab_test|
-  page.driver.set_cookie("ABTest-#{ab_test}", test_group)
+  page.driver.add_headers("GOVUK-ABTest-#{ab_test}" => test_group)
```

Comment:
I suspect the old version only worked on production, and the new version will only work on dev/integration/staging.

On production, the header will be overridden by the CDN because it has no cookie set.

I think we should set both and add a comment explaining why. It's probably safe to use both as long as the ab_testing.feature tests don't call this step - those tests really should be testing the whole stack, rather than setting intermediate A/B testing headers.